tests/test_endpoints/test_post.py

```diff
@@ -0,0 +1,262 @@
+from gramps_webapi.const import ENV_CONFIG_FILE, TEST_AUTH_CONFIG
+
+
+def get_headers(client, user: str, password: str) -> Dict[str, str]:
+    """Get the auth headers for a specific user."""
+    rv = client.post("/api/token/", json={"username": user, "password": password})
+    access_token = rv.json["access_token"]
+    return {"Authorization": "Bearer {}".format(access_token)}
+
+
+def make_handle() -> str:
+    """Make a new valid handle."""
+    return str(uuid.uuid4())
+
+
+class TestObjectCreation(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.name = "Test Web API"
+        cls.dbman = CLIDbManager(DbState())
+        cls.dbman.create_new_db_cli(cls.name, dbid="sqlite")
+        with patch.dict("os.environ", {ENV_CONFIG_FILE: TEST_AUTH_CONFIG}):
+            cls.app = create_app()
+        cls.app.config["TESTING"] = True
+        cls.client = cls.app.test_client()
+        sqlauth = cls.app.config["AUTH_PROVIDER"]
+        sqlauth.create_table()
+        sqlauth.add_user(name="user", password="123", role=ROLE_GUEST)
+        sqlauth.add_user(name="admin", password="123", role=ROLE_OWNER)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.dbman.remove_database(cls.name)
+
+    def test_objects_add_note(self):
+        """Add a single note via objects."""
+        handle = make_handle()
+        obj = [
+            {
+                "_class": "Note",
+                "handle": handle,
+                "text": {"_class": "StyledText", "string": "My first note."},
+            }
+        ]
+        headers = get_headers(self.client, "user", "123")
+        rv = self.client.post("/api/objects/", json=obj, headers=headers)
+        self.assertEqual(rv.status_code, 403)
+        headers = get_headers(self.client, "admin", "123")
+        rv = self.client.post("/api/objects/", json=obj, headers=headers)
+        self.assertEqual(rv.status_code, 201)
+        rv = self.client.get(f"/api/notes/{handle}", headers=headers)
+        self.assertEqual(rv.status_code, 200)
+
+    def test_add_note(self):
+        """Add a single note."""
+        handle = make_handle()
+        obj = {
+            "handle": handle,
+            "text": {"_class": "StyledText", "string": "My first note."},
+        }
+        headers = get_headers(self.client, "user", "123")
+        rv = self.client.post("/api/notes/", json=obj, headers=headers)
+        self.assertEqual(rv.status_code, 403)
+        headers = get_headers(self.client, "admin", "123")
+        wrong_obj = {"_class": "Person", "handle": handle}
+        rv = self.client.post("/api/notes/", json=wrong_obj, headers=headers)
+        self.assertEqual(rv.status_code, 400)
+        rv = self.client.post("/api/notes/", json=obj, headers=headers)
+        self.assertEqual(rv.status_code, 201)
+        rv = self.client.get(f"/api/notes/{handle}", headers=headers)
+        self.assertEqual(rv.status_code, 200)
```

Comment:
All other tests also verify the content to some extent. Was that intentionally left out here and in `test_objects_add_note`?

Reply:
No, not intentional!